### PR TITLE
Aw projects page

### LINKED ISSUE #46: unscored/scored projects page on judge side
replace this with fake api data

### PR DESCRIPTION
**Pull request by Warrenwwang1**
fixed #46  and #47

### CODE REVIEW

**Review by lajiang2 — DISMISSED** [review]
remove `server/node_modules` from pr

File: client/src/components/ProjectInfo/ProjectInfo.jsx [hunk 1]
```diff
@@ -8,7 +8,12 @@ import "./ProjectInfo.css";
 
 
 class ProjectInfo extends Component{
+    constructor(props) {
+        super(props);
+    }
+
     render() {
+        console.log("team: " + this.props.location.state.table);
```
Comment (juliawang18) [inline client/src/components/ProjectInfo/ProjectInfo.jsx:16]: what the frick is this

File: client/src/components/ScoringOverview/Project.jsx [hunk 2]
```diff
@@ -4,6 +4,7 @@ import { withRouter } from "react-router-dom";
 
 class Project extends Component {
     render() {
+        console.log("INSIDE PROJECT: " + this.props.key);
```
Comment (juliawang18) [inline client/src/components/ScoringOverview/Project.jsx]: WHAT THE FUK IS THIS

File: client/src/components/ScoringOverview/ScoringOverview.jsx [hunk 3]
```diff
@@ -69,9 +74,19 @@ class ScoringOverview extends Component {
                     </div>
                 </div>
             <ul>
+                {console.log(this.state.renderedList)}
```
Comment (juliawang18) [inline client/src/components/ScoringOverview/ScoringOverview.jsx]: this is NOT amateur hours

**Review by juliawang18 — DISMISSED** [review]
u suck

**Review by partyshah — APPROVED** [review]
nice

**Review by partyshah — APPROVED** [review]
nice

**Pull request merged**